packages/esm-appointments-app/src/home-appointments/appointments-list.component.tsx

```diff
@@ -212,50 +202,73 @@ const AppointmentsBaseTable = () => {
   }
 
   return (
-    <div className={styles.homeAppointmentsContainer}>
-      <div className={styles.headerContainer}>
-        <div className={!isDesktop(layout) ? styles.tabletHeading : styles.desktopHeading}>
-          <h4>{t('todaysAppointments', "Today's Appointments")}</h4>
+    <>
+      <div className={styles.homeAppointmentsContainer}>
+        <div className={styles.headerContainer}>
+          <div className={!isDesktop(layout) ? styles.tabletHeading : styles.desktopHeading}>
+            <h4>{t('todaysAppointments', "Today's Appointments")}</h4>
+          </div>
+          <div className={styles.actionLinks}>
+            <ViewCalendarLink />
+            <AddAppointmentLink />
+          </div>
         </div>
-        <AddAppointmentLink />
+        <DataTable
+          data-floating-menu-container
+          headers={tableHeaders}
+          overflowMenuOnHover={isDesktop(layout) ? true : false}
+          rows={tableRows}
+          size={isDesktop(layout) ? 'xs' : 'md'}
+          useZebraStyles={appointments?.length > 1 ? true : false}>
+          {({ rows, headers, getHeaderProps, getTableProps, getRowProps }) => (
+            <TableContainer className={styles.tableContainer}>
+              <Table {...getTableProps()} className={styles.appointmentsTable}>
+                <TableHead>
+                  <TableRow>
+                    {headers.map((header) => (
+                      <TableHeader {...getHeaderProps({ header })} key={header.id}>
+                        {header.header}
+                      </TableHeader>
+                    ))}
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {rows.map((row, index) => {
+                    return (
+                      <React.Fragment key={row.id}>
```

Comment:
This doesn't need to be a React.Fragment. You can just return TableRow directly (and pass it the key prop)